src/builder/guard.rs

```diff
@@ -109,3 +138,37 @@ impl<'a> Guard<'a> {
         return Ok(());
     }
 }
+
```

Comment:
I need a comment what `true`/`false` in keep_rel_path means. Or make two sets with more specific names.